src/packages/@app/pages/Help/HelpPage.ViewModel.ts

```diff
@@ -547,6 +592,13 @@ class HelpPageViewModel {
     }
   };
 
+  public listVoteUsingPostId = async (postID: string) => {
+    if (postID) {
+      const result = await this.cannyService.listVotesUsingPostId(postID);
+      return result;
+    }
+  };
+
   public listChangeLog = async (type: string) => {
     const result = await this.cannyService.listChangeLog(type);
     return result;
```

Comment:
add jsdoc comment

Reply:
Is there already a function that upvotes using postId

Reply:
Resolved